src/config.ts

```diff
@@ -154,5 +161,15 @@ export const paraTimesConfig: ParaTimesConfig = {
   [ParaTime.Sapphire]: sapphireConfig,
 }
 
+export const getRuntimeConfig = (runtimeId: string) => {
+  const runtimeConfig = Object.values(paraTimesConfig).find(
+    config => config.mainnet.runtimeId === runtimeId || config.testnet.runtimeId === runtimeId,
+  )
+  if (!runtimeConfig) {
+    throw new Error(`Runtime config not found for runtimeId: ${runtimeId}`)
+  }
+  return runtimeConfig
+}
```

Comment:
This is yucky. We shouldn't copy it into the good codebase 

We should pass a fuller runtime configuration from
https://github.com/oasisprotocol/oasis-wallet-web/blob/eb4639f437b8c51364dc7b477c2c8d51b57d1851/src/app/state/paratimes/saga.ts#L75-L79
through
- submitParaTimeTransaction
- prepareParatimeTransfer
- OasisTransaction.buildParaTimeTransfer